### PR TITLE
fix: Use BODY_LIMIT when creating app

### PR DESCRIPTION
**Pull request by jgoz**
## In this PR:

- Bug fix (non-breaking change which fixes an issue)

The BODY_LIMIT env variable was being used in the content type parser, but not when creating the fastify app. This meant the practical body limit could not be raised past 50MB.

This PR uses the BODY_LIMIT var for both values.

### Checklist:

* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/ducktors/turborepo-remote-cache/pulls) for the same update/change?

* [x] Have you lint your code with `pnpm lint` locally prior to submission?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you successfully ran build with `pnpm build` of your changes locally?
* [x] Have you successfully ran tests with `pnpm test` of your changes locally?
* [x] Have you commit using [Conventional Commits](https://github.com/ducktors/turborepo-remote-cache#how-to-commit)?

### CODE REVIEW

**Comment by jgoz** [comment]
@matteovivona Does this change make sense?

**Comment by jakeinflab** [comment]
I agree with this change. We are running a very large project where our cache size exceeds 50MB, resulting in issues with loading the cache. Could you please review this change?
@matteovivona @jgoz

**Comment by matteovivona** [comment]
@jgoz thank you for your PR! It makes sense. 
Could you also update the documentation with this new variable? I would really appreciate it!

File: src/index.ts [hunk 1]
```diff
@@ -7,9 +7,7 @@ const app = createApp({
   trustProxy: true,
   // Default is 1MB
   // https://fastify.dev/docs/latest/Reference/Server/#bodylimit
-  // Here, we raise it to 50 MB (50 * 1024 * 1024).
-  // number is measured in Bytes
-  bodyLimit: 52428800,
+  bodyLimit: env.get().BODY_LIMIT,
```
Comment (matteovivona) [inline src/index.ts:10]: include this new variable in the docs

**Comment by matteovivona** [comment]
@all-contributors please add @jgoz for code and doc

**Review by fox1t — APPROVED** [review]
LGTM after doc change. Thanks for the PR.

File: src/index.ts (hunk 1 above)
Comment (jgoz) [reply src/index.ts:10]: This variable already exists in the docs, it just wasn't being used in this one spot

**Comment by jgoz** [comment]
@fox1t @matteovivona The docs already have this variable listed, it just wasn't being used in this spot.

**Pull request merged**